src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (firstValue) => {
```

Comment:
When your arrow function has one parameter you don't actually need the round brackets.
`return firstValue => {`